.github/workflows/benchmarking-regression.yml

```diff
@@ -49,6 +49,7 @@ jobs:
           ref: main
 
       - name: Install dependancies
+        if: steps.cached-pip-wheels.outputs.cache-hit != 'true'
```

Comment:
anything but `true` is False ?

Reply:
Yeah if cache-hit is true (makes the expression false) then we do not want to install dependencies because they have been imported in by a prior step. The step is skipped. If cache-hit is false (makes this expression true) then install dependencies.